neurostuff/tests/api/test_openapi.py

```diff
@@ -0,0 +1,18 @@
+import schemathesis
+from hypothesis import settings
+import pytest
+
+#from ...core import app
+
+
+# schema = schemathesis.from_wsgi("/api/openapi.json", app)
+
+# @pytest.mark.skip(reason="Currently cannot get these tests to work")
```

Comment:
Hi! My name is Dmitry, and I am the author of Schemathesis :) Just letting you know that if you need any help with integrating it, I'd be happy to help :)

Reply:
Thank you! I'll open a separate pull request to get your help on using schemathesis